src/internal/components/pluggableVisualizations/heatMap/tests/PluggableHeatmap.spec.tsx

```diff
@@ -87,6 +87,54 @@ describe("PluggableHeatmap", () => {
         });
     });
 
+    it("heatmap should transform location item to rows bucket", async () => {
```

Comment:
as I see in assertion, there is no row bucket
do you mean `"should transfer location bucket item to stack bucket item after switching to heatmap"`?

Reply:
No. it mean that location bucket items should be included when finding item for rows bucket.
see [here](https://github.com/gooddata/gooddata-react-components/pull/1475/files#diff-ff8ace3270a4778d08943ed7f8a44f09R57)